UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/DistributionService.cs

```diff
@@ -41,5 +53,55 @@ public static bool SaveDistributorRequest(CustomCloudEvent cloudEvent)
                 return s_recordDistributorRequestQueue.ToList();
             }
         }
+
+        public bool SaveCommandDistributorRequest(string subject, HttpStatusCode? statusCode)
+        {
+            try
+            {
+                CommandDistributionRequest? commandDistributorRequest = s_CommandDistributionList.Where(a => a.Subject == subject).LastOrDefault();
```

Comment:
Please simplify LINQ expression as below:
```suggestion
                CommandDistributionRequest? commandDistributorRequest = s_CommandDistributionList.LastOrDefault(a => a.Subject == subject);
```

Reply:
Fixed